packages/panna-sdk/README.md

```diff
@@ -189,6 +195,30 @@ Wallet management utilities provide tools for user authentication, allowing user
 - `linkAccount`: Connects a new profile to the current user.
 - `unlinkAccount`: Disconnects an existing profile from the current user.
 
+### Onramp (Fiat-to-Crypto)
+
+The onramp module enables fiat-to-crypto conversions through multiple payment providers. It handles the complete purchase flow from session preparation to payment confirmation.
+
+#### Key Functions
+
+- `getOnrampProviders`: Returns available payment providers for a given country.
+- `getTokenFiatPrices`: Fetches current fiat prices for tokens.
+
+The `onRampPrepare` and `onRampStatus` methods are internal and not part of the public API. Please use the `getOnrampProviders` and `getTokenFiatPrices` methods for onramping functionality.
```

Comment:
Is this true?
- these methods are used in docs examples throughout `core/onramp/README.md`
- they are exported from the thirdweb subdirectory